lang/pt_BR.json

```diff
@@ -13,21 +13,21 @@
     "Close": "Fechar",
     "More pages": "Mais páginas",
     "Subset": "Subconjunto",
-    "Disclaimer about ROMs": "Aviso sobre ROMs",
+    "Disclaimer about ROMs": "Aviso sobre as ROMs",
     "RetroAchievements.org does not condone or supply any copyright-protected ROMs to be used in conjunction with the supplied emulators.": "RetroAchievements.org não endossa nem fornece ROMs protegidas por direitos autorais para serem usadas em conjunto com os emuladores fornecidos.",
-    "There are no copyright-protected ROMs available for download on RetroAchievements.org.": "Não há ROMs protegidas por direitos autorais disponíveis para download no RetroAchievements.org.",
+    "There are no copyright-protected ROMs available for download on RetroAchievements.org.": "Não há ROMs protegidas por direitos autorais disponíveis para baixar no RetroAchievements.org.",
     ":earned of :total achievements unlocked": ":earned de :total conquistas desbloqueadas",
     ":earned of :total softcore achievements unlocked": ":earned de :total conquistas softcore desbloqueadas",
-    ":earned of :total points earned": ":earned de :total pontos ganhos",
-    ":earned of :total softcore points earned": ":earned de :total pontos softcore ganhos",
-    "Finished": "Finalizado",
+    ":earned of :total points earned": ":earned de :total de pontos ganhos",
+    ":earned of :total softcore points earned": ":earned de :total de pontos softcore ganhos",
+    "Finished": "Zerado",
```

Comment:
This may not be right. This label may appear on "games" that aren't really games, such as events. In other words, it can't really be beaten or mastered. Given that context, should "Zerado" still be used?